Copy to SD Card root directory to update/config.ini

```diff
@@ -102,6 +102,16 @@ serial_port:P1:6 P2:0 P3:0 P4:0
 #   Options: [disable: 0, enable: 1]
 emulated_m600:1
 
+#### Emulated M109 And M190
+# The TFT intercepts the blocking M109 and M190 gcodes (set target hotend and bed temperatures)
+# and converts them to the non blocking M104 and M140 gcodes respectively.
+#
+# NOTE: Enable it to allow the TFT can continue to dialog (e.g. using menus sending gcodes to the
```

Comment:
"to allow the TFT can continue" - typo?

Reply:
hmm no. it should be correct. Do you mean wrong syntax?

Reply:
Hard to explain, the whole sentence sounds wrong. My English is not that bad but it's hard for me to understand it (ex. "to dialog" is not a verb in English). Only by checking the code I have some idea what it supposed to mean.
IDK, maybe "Enable it so the TFT can still communicate with Marlin firmware even if the target temperature is not reached yet."

Reply:
Later edit:
Maybe add: "otherwise communication will be frozen until desired/set temps are obtained."

Reply:
yes I agree, thanks. Using your comments